ansible_builder/main.py

```diff
@@ -49,18 +52,55 @@ def build_command(self):
             self.build_context
         ]
 
+    def run_in_container(self, command, **kwargs):
+        wrapped_command = [self.container_runtime, 'run','--rm']
+
+        wrapped_command.extend(['-v', f"{os.path.abspath(self.build_context)}:/context:Z"])
+
+        wrapped_command.extend([self.tag, '/bin/bash', '-c', ' '.join(command)])
+
+        return run_command(wrapped_command, **kwargs)
+
+    def run_intermission(self):
+        run_command(self.build_command, capture_output=True)
+
+        rc, introspect_output = self.run_in_container(
+            ['python3', '/context/introspect.py'], capture_output=True
+        )
+        collection_data = yaml.safe_load('\n'.join(introspect_output))
+
+        # Add data from user definition, go from dicts to list
+        collection_data['system']['user'] = self.definition.user_system
+        collection_data['python']['user'] = self.definition.user_python
+        system_lines = ansible_builder.introspect.simple_combine(collection_data['system'])
+        python_lines = sanitize_requirements(collection_data['python'])
+
+        bindep_output = []
+        if system_lines:
+            write_file(os.path.join(self.build_context, BINDEP_COMBINED), system_lines + [''])
+
+            rc, bindep_output = self.run_in_container(
+                ['bindep', '-b', '-f', '/context/{0}'.format(BINDEP_COMBINED)],
+                allow_error=True, capture_output=True
+            )
```

Comment:
hmm.... should we _delete_ the `BINDEP_COMBINED` file here? I learn towards "no" but feel like I have to ask the question.